quill-async-postgres/src/test/scala/io/getquill/context/async/postgres/ArrayExtraAsyncEncodingSpec.scala

```diff
@@ -0,0 +1,56 @@
+package io.getquill.context.async.postgres
+
+import java.time.{ ZonedDateTime }
+
+import io.getquill.context.sql.encoding.ArrayEncodingExtraSpec
+import org.joda.time.{ DateTime => JodaDateTime }
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class ArrayExtraAsyncEncodingSpec extends ArrayEncodingExtraSpec {
```

Comment:
If I append these changes to existing (initially that was like that) , that will break tests in other places - tests in cassandra,  etc. Those base trait were used everywhere.  But ZonedDateTime were added only to async and jdbc driver. Otherwise it will require all contexts support ZonedDateTime.